dist/react-number-format.min.js

```diff
@@ -3,4 +3,4 @@
  * Author : Sudhanshu Yadav
```

Comment:
Is it necessary to keep build version in source code? I guess it is because we don't have build/publish process yet?